### ISSUE TITLE
car driving model

### ISSUE THREAD

**Issue by chenmin1107** [ros-simulation/stage_ros#48]
Hi there,

What is non-honolomic car driving model used in stage_ros? If I send a msg (cmd_vel), how would stage_ros move the vehicle around in the simulator?

My guess is Ackermann car model (https://en.wikipedia.org/wiki/Ackermann_steering_geometry). Is that correct?

**Comment by dkargin** [comment]
Nope. It is pure kinematic model that is used in libstage/stage_ros. You can control its velocity immediately.

In fact, there are three control modes in libstage: 
```
typedef enum { CONTROL_ACCELERATION, CONTROL_VELOCITY, CONTROL_POSITION } ControlMode; 
```
 - CONTROL_VELOCITY - you change robot state by setting it velocity directly (x, y, angular).
 - CONTROL_ACCELERATION - you set acceleration, and velocity and position are updated accordingly
 - CONTROL_POSITION - you change position of the robot directly.

stage_ros uses CONTROL_VELOCITY. You can use intermediate node, that recalculates from ackerman_msgs to appropriate sensor_msgs::Twist to make robot movement look exactly like ackemann's kinematics.
